spec/support/sample/test_shard.yml

```diff
@@ -0,0 +1,43 @@
+name: test_app
+version: 0.1.0
+
+authors:
+  - Amber <contact@ambercr.io>
+
+crystal: 0.23.1
+
+license: UNLICENSED
+
+targets:
+  test_app:
+    main: src/test_app.cr
+
+  amber:
+    main: lib/amber/src/amber/cli.cr
+
+development_dependencies:
+  amber_spec:
+    github: amber-crystal/amber-spec
+
+dependencies:
+  amber:
+    path: ../../amber
```

Comment:
This test_shard.yml file is used to run tests in the current amber context. 

With this file in place, we can replace the generated **shard.yml** for the `test_app`, causing the tests to look at the current amber context (very useful for testing changes to the framework to make sure the CLI does not break).